src/atoms/buttons/Button.js

```diff
@@ -8,34 +8,44 @@ function Button ( props ) {
     children,
     onClick,
     shake,
-    className
+    className,
+    buttonType,
+    disabled
   } = props;
 
   return (
-    <div
+    <button
       className={ classnames(className, styles[type], { [styles.shaking]: shake }) }
```

Comment:
For the select card, Josh suggested we move towards explicit type props rather than having to pass in a type string. I came up with a decent solution for the select card, I think this is something we need to standardize